tests/test_cli.py

```
@@ -0,0 +1,877 @@
+
+            assert len(install_called) == 1
+
+
+class TestMainFunction:
+    """Test the main CLI function."""
+
+    def test_main_run_bot(self):
+        """Test running the bot when config exists."""
+
+        # ✅ CORRECT: Use explicit function replacement (mmrelay pattern)
+        def mock_exists(path):
```

Ruff (0.12.2)

Unused function argument: `path`

(ARG001)